workspace-cache-server/README.md

```diff
@@ -0,0 +1,43 @@
+# workspace-cache-server
+
+> :herb: This is the cache server workspaces :octocat: All you need in a nut:shell:
+
+This template is a simple express API using a postgresql database. To run this example you need a postgresql database and create the tables. The tables for this example are at *start.sql* file.
+
+In case you want or need to interact with the database, [here](https://wiki.postgresql.org/wiki/PostgreSQL_Clients) is a full list of clients.
+
+This basic example is written is typescript.
+
+## Installation
+
+Use [yarn](https://yarnpkg.com) to install dependencies.
+
+```bash
+$ yarn
+```
+
+## Usage
+
+### Start a database
+
+To keep everything simple, we recommend you to start  postgresql database, using [docker](https://www.docker.com/). If you already have, follow the instructions [here](https://hub.docker.com/_/postgres) to start a local postgresql database.
+
+Before you start, you also need to create the tables. To do so, at the root of this project you can find a *start.sql* file. Depending on the client you are using, either run that file, or copy/paste and execute the commands. It will create the base database and the table. Without this, the cache will not work.
+
+Also, it is necessary to rename the file *.env.example* to *.env* and fill the values.
+
```

Comment:
And then what? Any other `yarn` command (`yarn start` crashes)? Do we run anything on the `workspace-blockchain` directory? I can run `yarn test` in the `workspace-blockchain` directory, but nothing is reflected in the setvalue table.

```
alberto@alberto-laptop:~/Code/starter-kit/workspace-cache-server$ yarn start
yarn run v1.19.1
warning package.json: No license field
$ ts-node dist/index.js
Error: Cannot find module '/home/alberto/Code/starter-kit/workspace-cache-server/dist/index.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:741:12)
    at main (/home/alberto/Code/starter-kit/workspace-cache-server/node_modules/ts-node/src/bin.ts:212:14)
    at Object.<anonymous> (/home/alberto/Code/starter-kit/workspace-cache-server/node_modules/ts-node/src/bin.ts:470:3)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
https://github.com/HQ20/starter-kit/pull/275/files#diff-748a63577c3070b096aed1c762f39f8dR33

`yarn start` is for compiled code. Use `yarn dev` to run it without compiling.

Reply:
It is not very clear what is the difference between "starts the server in dev mode" and "runs the compiled code".

Is it `yarn dev` the same as `yarn build && yarn start`?

Reply:
yes,
I'll think of some different commands